.idea/misc.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The changes in this file should be reverted, since it isn't related to the bug being fixed.